src/Microsoft.Extensions.Configuration.EnvironmentVariables/project.json

```diff
@@ -6,7 +6,7 @@
     "nowarn": [ "CS1591" ],
     "xmlDoc": true
   },
-  "description": "Environment variables configuration provider implementation of the Microsoft.Extensions.Configuration.IConfigurationProvider interface.",
+  "description": "Environment variables configuration provider implementation of Microsoft.Extensions.Configuration.",
```

Comment:
Reads a bit strange. Maybe "for Microsoft.Extensions.Configuration"  instead of "of"?